packages/api-explorer/package.json

```
@@ -17,10 +17,11 @@
   },
   "scripts": {
     "test": "jest",
+    "bundle": "tsc && webpack --config webpack.prod.config.js",
```

I'd probably go with `build` here :)

```suggestion
    "build": "tsc && webpack --config webpack.prod.config.js",
```